Vagrantfile

```diff
@@ -9,11 +9,16 @@ $script = <<SCRIPT
 
     # Install system requirements
     apt-get update -y
-    apt-get install -y git python2.7 python-pip python2.7-dev firefox dbus-x11 vim libjpeg-dev libxml2-dev libxslt1-dev
-    apt-get install -y libtiff4-dev libjpeg8-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev tcl8.5-dev tk8.5-dev python-tk
+    apt-get install -y xorg git python2.7 python-pip python2.7-dev dbus-x11 vim libjpeg-dev libxml2-dev libxslt1-dev
```

Comment:
We also use a different version of git from a ppa. 

We should be sure that we can compile pillow with jpg support on this box.

Other than that, let's just add this info to the ticket. I don't want to spend cycles perfecting until it's clear we'll be using it long-term.

EDIT: I see libjpeg8, which means pillow will compile fine. No need to check, but leaving the comment here just to pour more bits of data on you that may come in handy later. Pillow will successfully compile when there are system libs missing; it'll just not-support that area of its functionality. So, it can get tricky on you.